src/tests/functional/plugin/gpu/shared_tests_instances/behavior/ov_plugin/life_time.cpp

```diff
@@ -16,4 +16,12 @@ namespace {
                                         //CommonTestUtils::DEVICE_BATCH,
                                         "HETERO:GPU"),
             OVHoldersTest::getTestCaseName);
+
+    INSTANTIATE_TEST_SUITE_P(smoke_VirtualPlugin_BehaviorTests, OVHoldersTestWithConfig,
+                    ::testing::Values("AUTO:GPU,CPU",
+                                        "AUTO:CPU,GPU",
+                                        "MULTI:GPU",
+                                        "MULTI:GPU,CPU",
+                                        "MULTI:CPU,GPU"),
```

Comment:
what if CPU is not available? E.g. `ENABLE_INTEL_CPU=OFF`? do we need CPU here or `AUTO:GPU` is enough?